doc/source/parameters/resolved_cfd-dem/resolved_cfd-dem.rst

```diff
@@ -1,3 +1,91 @@
+		subsection gravity
+			set Function expression =0;0;0
+		end
+		subsection particle info 0
+			set density    = 1
+			subsection position
+				set Function expression =0;0;0
+			end
+			subsection velocity
+				set Function expression =0;0;0
+			end
+		    	subsection omega
+		    		set Function expression =0;0;0
+		    	end
+		    	set inertia    = 1
+		    	set pressure x = 0
+		    	set pressure y = 0
+		    	set pressure z = 0
+		    	set radius     = 0.2
+	end
+	
+* The ``number of particles`` Number of particles reprensented by IB.
+
+* The ``stencil order`` parameter control the order of the Lagrange polynomial used to imposed the sharp interface immersed boundary condition. The order of the stencil should be equal or higher then the order of the FEM scheme.
+
+* The ``length ratio`` parameter controls the length of the zone used to define the Lagrange polynomial. The length ratio should be kept as small as possible and above 1. A good starting value is two times the average aspect ratio of the elements in the mesh multiplied by the order of the FEM scheme.
+
+* The ``assemble Navier-Stokes inside particles`` parameter determined if the Navier-Stokes equations are solved inside the particle or not. If the Navier Stokes equation is not solved ( the parameter is false ), the solver will solve a Poisson equation for each variable in the problem. This eliminates the need to define a reference value for the pressure. 
+
+* The ``calculate force`` parameter control if the force is evaluated on each particle. 
+
+* The ``ib force output file`` parameter is the file's name where the derivated value associated with each particle is stored. One file will be created for each particle in the simulation.
+
+* The ``ib particles pvd file`` parameter is the file's name that will be created to animate the particles. This file stores all the derivated values calculated for each of the particles. This file is compatible with Paraview 
+
+* The ``initial refinement`` parameter control how many time the refinement zone around each of the particle is applied before the simulation start. Each application of the refinement zone reduce the size of the element by a factor of two.
+
+* The ``refine mesh inside radius factor`` parameter defined the inside radius of the hyper shell that forms the refinement zone around the particle. The radius used is the product between this factor and the particle's radius. 
+
+* The ``refine mesh outside radius factor`` parameter defined the outside radius of the hyper shell that forms the refinement zone around the particle. The radius used is the product between this factor and the particle's radius. 
+
+* The ``integrate motion`` parameter control if the dynamics equations of the particles are calculated. If this parameter is set to false, the dynamics of the particle will not be calculated. The particle's position and velocity will only be defined by the particles' position and velocity function.
```

Comment:
```suggestion
* The ``integrate motion`` parameter controls if the dynamics equations of the particles are calculated. If this parameter is set to false, the particles remain static.  If ``ìntegrate motion=true`` the position and the velocity will be defined by the particles' position and velocity function.
```

Reply:
It might be a good idea to define with a .. note:: what is the dynamics equation of the particles exactly.

Reply:
I would let this part go in another PR with the Contact equation and the relaxation schemes used. After the Merge of the Contact PR.